bench/formal/mor1kx_execute_alu.sby

```diff
@@ -1,35 +1,36 @@
 [tasks]
 #Tags are not used in code but specify type of parameters set.
-test1  serial_mul        serial_div       serial_shift
-test2  pipelined_mul     simulation_div   barrel_shift
-test3  threestage_mul    none  _div       serial_div
-test4  simulation_mul    simulation_div   barrel_shift
+test_serial      serial_mul        serial_div       serial_shift
+test_pipe_mul    pipelined_mul     simulation_div   barrel_shift
+test_3stage_mul  threestage_mul    no_div           serial_shift
+test_sim_mul     simulation_mul    simulation_div   barrel_shift
 
+# Depth set to 64 to handle 32 cycle serial operations
 [options]
 mode prove
-depth 20
+depth 64
```

Comment:
Maybe we can add depth 64 only for test_serial.  For other tasks, we can reduce depth.

Reply:
Good idea I didn't know about that option